.isort.cfg

```diff
@@ -1,3 +1,7 @@
 [settings]
 known_third_party = asv,click,clickhouse_driver,dateutil,google,graphviz,impala,jinja2,kudu,multipledispatch,numpy,pandas,pkg_resources,plumbum,psycopg2,pyarrow,pydata_google_auth,pygit2,pymapd,pymysql,pyspark,pytest,pytz,regex,requests,ruamel,setuptools,sphinx_rtd_theme,sqlalchemy,toolz
 ensure_newline_before_comments=true
+line_length = 79
```

Comment:
hmm, I think these should all be in setup.cfg which is the common place, why are we moving here?

Reply:
I moved that because isort was already using that .. so just to keep all configurations in just one file

but I check and setup.cfg also is supported: https://github.com/asottile/seed-isort-config/blob/master/seed_isort_config.py#L19

I will try to move all this to setup.cfg